cfgov/v1/jinja2/v1/_includes/organisms/image-and-text-50-50.html

```diff
@@ -1,20 +1,63 @@
-
 {# ==========================================================================
 
-   image_and_text_50_50.render()
+   image_and_text-50-50.render()
```

Comment:
Should be `image_and_text_50_50` as shown in https://github.com/cfpb/cfgov-refresh/pull/1063/files#diff-9c2c1b680ebe958baab039dca0c34f45R22